internal/test/framework/windowsvm.go

```diff
@@ -49,9 +49,10 @@ type WindowsVM interface {
 	// CopyFile copies the given file to the remote directory in the Windows VM. The remote directory is created if it
 	// does not exist
 	CopyFile(string, string) error
-	// RetrieveFiles retrieves the list of file from the directory in the remote Windows VM to the local host. As of
-	// now, we're limiting every file in the remote directory to be written to single directory on the local host
-	RetrieveFiles(string, string) error
+	// RetrieveDirectories retrieves the list of files and directories from the directory in the remote Windows VM to
```

Comment:
`retrieves the list of files and directories` This is saying that the function serves the same function as a remote `ls`.

This comment can be reworded as
```
RetrieveDirectories recursively copies the files and directories from the directory in the remote Windows VM
to the given directory on the local host.
```